mypyc/test-data/irbuild-str.test

```diff
@@ -159,3 +159,20 @@ L2:
 L3:
     unreachable
 
+[case testStrFormatMethod]
```

Comment:
Add irbuild tests for skipping `PyObject_Str` for `str` values, skipping empty strings and `{{` / `}}`.